dist/calendar/CalendarMonth.js

```diff
@@ -65,7 +60,33 @@ var CalendarMonth = _react2['default'].createClass({
     highlightedRange: _react2['default'].PropTypes.object,
     onMonthChange: _react2['default'].PropTypes.func,
     onYearChange: _react2['default'].PropTypes.func,
-    value: _utilsCustomPropTypes2['default'].momentOrMomentRange
+    value: _utilsCustomPropTypes2['default'].momentOrMomentRange,
+    locale: _react2['default'].PropTypes.string
+  },
+
+  _setupLocale: function _setupLocale(locale) {
+    this.moment = (0, _moment2['default'])();
+
+    this.moment.locale(locale);
+
+    var lang = this.moment.localeData(locale);
+
+    this.WEEKDAYS = _immutable2['default'].List(lang._weekdays).zip(_immutable2['default'].List(lang._weekdaysShort));
```

Comment:
@thaerlabs actually its not `ru` locale problem. Moment supports 2 way month localization:
1) array of month names, like `en`
```
_months: ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"]
```
2) object with months names in several forms, like `ru`, `ka`, `be`, `lt` and [others](https://github.com/moment/moment/search?utf8=%E2%9C%93&q=standalone)
```
_months : {
   format: 'января_февраля_марта_апреля_мая_июня_июля_августа_сентября_октября_ноября_декабря'.split('_'),
   standalone: 'январь_февраль_март_апрель_май_июнь_июль_август_сентябрь_октябрь_ноябрь_декабрь'.split('_')
}
```
Here how it works [inside Moment](https://github.com/moment/moment/blob/602718d67f06011cf6483387ad96989528f55597/src/lib/units/month.js#L65) 
```
export function localeMonths (m, format) {
    return isArray(this._months) ? this._months[m.month()] :
        this._months[MONTHS_IN_FORMAT.test(format) ? 'format' : 'standalone'][m.month()];
}
```
Just use same logic in your code